.changeset/hungry-mayflies-destroy.md

```diff
@@ -0,0 +1,6 @@
+---
+"@hashicorp/design-system-components": minor
+"@hashicorp/design-system-tokens": minor
+---
+
+Added global tokens for border radius
```

Comment:
@alex-ju Is this changeset ok? I wasn't positive if I should include "@hashicorp/design-system-components" although I did update CSS to make use of the new design tokens.

Reply:
Thanks!